stable_baselines/common/policies.py

```diff
@@ -434,7 +440,10 @@ def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, reuse=Fals
             if feature_extraction == "cnn":
                 pi_latent = vf_latent = cnn_extractor(self.processed_obs, **kwargs)
             else:
-                pi_latent, vf_latent = mlp_extractor(tf.layers.flatten(self.processed_obs), net_arch, act_fun)
+                pi_acts, vf_acts = mlp_extractor(tf.layers.flatten(self.processed_obs),
+                                                 net_arch, act_fun)
+                self.ff_out = {'value': vf_acts, 'policy': pi_acts}
+                pi_latent, vf_latent = pi_acts[-1], vf_acts[-1]
 
             self.value_fn = linear(vf_latent, 'vf', 1)
             self.ff_out = {'value': [vf_latent], 'policy': [pi_latent]}
```

Comment:
This is overwriting the previous `self.ff_out`?

Reply:
oops yeah I'll remove that. Should I maybe have a default value of self.ff_out, i.e. `None` that I assign to it beforehand?

Reply:
I'd suggest moving this assign to the first `if` branch (for if you call `cnn_extractor` instead)?  `None` would be awkward as you'd need to have users of this code special-case that.

Reply:
yeah that makes sense.